### PR TITLE
docs(changelog): add section for 5.0.0 release

### PR DESCRIPTION
**Pull request by kapil-agnihotri**
ING-3370

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,48 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
...
+- Support to configure details on HTTP client max connections and connection eviction via environment variables
+- Support collecting metrics on Request class HTTP clients 
+- Support for Protocol Buffer File as a format for exporting data
+- Support for TopoJSON as a format for exporting transformed data
+- Support to get list of Shapefile names
+- Support for XPlanGML 5.4 and XPlanGML 6.0 schema preset
+- Support to allow excluding root properties from structural Retype
+- Support to automatically add the codespace for INSPIRE feature types
+- Support for `xs:all` to the XML Schema Reader
+- Support to export data in Json format. GeoJson format to follow RFC7946 SPEC
```
Comment (thorsten-reitz) [inline CHANGELOG.md:22]: Wasn't this present before already?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,48 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
...
+- Support to read CRS information from Geopackage file rather than reading it from the schema
+- Enhanced support for loading XML/GML schema resembling Geopackage structure so that no information is lost
+- Support to specify leniency when using DateExtraction function 
+- Support to skip n-lines when reading csv and xls files
+ 
+### Changed
+- Merge schemas for HSD export as XML or Json to one schema to work around the restriction that only single schema can be imported from HSD file
+
+### Removed
+- Deegree functionality due to Java 17 migration
```
Comment (thorsten-reitz) [inline CHANGELOG.md:34]: Ì would reword to "deegree workspace configuration generation has been removed since the required libraries are not compatible with Java 17."

File: CHANGELOG.md (hunk 1 above)
Comment (kapil-agnihotri) [reply CHANGELOG.md:22]: Yes, I added this thinking to highlight RFC7946 SPEC because I guess before exported Json or GeoJson were not validated properly. 
I will remove this line it will confuse the readers.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,47 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
...
+- Support to read CRS information from Geopackage file rather than reading it from the schema
+- Enhanced support for loading XML/GML schema resembling Geopackage structure so that no information is lost
+- Support to specify leniency when using DateExtraction function 
+- Support to skip n-lines when reading csv and xls files
+ 
+### Changed
+- Merge schemas for HSD export as XML or Json to one schema to work around the restriction that only single schema can be imported from HSD file
+
+### Removed
+- Deegree workspace configuration generation has been removed since the required libraries are not compatible with Java 17
```
Comment (stempler) [inline CHANGELOG.md:33]: I would write

"Deegree workspace configuration generation has been removed since it requires further work to integrate updated dependencies that are compatible with Java 17"

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,47 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
 
 ## [Unreleased]
 
+## [5.0.0]
+
+### Added
+- Support for Java 17
+- Support for Groovy 2.5
+- Use jitpack for verifying artifacts
```
[the remark this answers is not in the document]
Comment (kapil-agnihotri) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,47 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
...
+- Support to configure details on HTTP client max connections and connection eviction via environment variables
+- Support collecting metrics on Request class HTTP clients 
+- Support for Protocol Buffer File as a format for exporting data
```
[the remark this answers is not in the document]
Comment (kapil-agnihotri) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,47 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
...
+- Support for TopoJSON as a format for exporting transformed data
```
[the remark this answers is not in the document]
Comment (kapil-agnihotri) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,47 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
...
+- Support to get list of Shapefile names
+- Support for XPlanGML 5.4 and XPlanGML 6.0 schema preset
+- Support to allow excluding root properties from structural Retype
+- Support to automatically add the codespace for INSPIRE feature types
+- Support for `xs:all` to the XML Schema Reader
+- Support to specify initial value during "Generate sequential ID" function
+- Support to preserve original order of columns during CSV export
```
[the remark this answers is not in the document]
Comment (kapil-agnihotri) [reply CHANGELOG.md]: Updated.

File: CHANGELOG.md (hunk 3 above)
Comment (kapil-agnihotri) [reply CHANGELOG.md]: Done.

**Review by thorsten-reitz — APPROVED** [review]
